servers/rendering/renderer_rd/forward_clustered/scene_shader_forward_clustered.cpp

```diff
@@ -479,6 +484,13 @@ void SceneShaderForwardClustered::ShaderData::_create_pipeline(PipelineKey p_pip
 
 	sc.constant_id = 1;
 	sc.int_value = p_pipeline_key.shader_specialization.packed_1;
+	sc.type = RD::PIPELINE_SPECIALIZATION_CONSTANT_TYPE_INT;
+	specialization_constants.push_back(sc);
+
+	sc = {}; // Sanitize value bits. "bool_value" only assigns 8 bits and keeps the remaining bits intact.
+	sc.constant_id = 2;
+	sc.bool_value = p_pipeline_key.emulate_point_size;
+	sc.type = RD::PIPELINE_SPECIALIZATION_CONSTANT_TYPE_BOOL;
```

Comment:
There seems to be plenty of bits available on "packed_1", so I'd just add this flag there instead of adding a new value to the push constant.